test/support/factory.ex

```diff
@@ -23,10 +23,20 @@ defmodule Wanda.Factory do
     %Catalog.Check{
       id: UUID.uuid4(),
       name: Faker.StarWars.character(),
+      group: Faker.Company.name(),
+      description: Faker.Lorem.sentence(),
+      remediation: Faker.Lorem.sentence(),
+      metadata: %{
+        target_type: Faker.StarWars.character(),
```

Comment:
If you want to be picky, you could get these values from arrays as well:
```target_type: Enum.random(["cluster", "host])` (check the values maybe in the web project.
The same for `cluster_type`

Reply:
Improved :D